src/group-constructors.js

```diff
@@ -55,32 +55,53 @@
     }
   };
 
-  scope.newUnderlyingPlayerForGroup = function(group) {
-    var underlyingPlayer;
+  scope.newUnderlyingAnimationForGroup = function(group) {
+    var underlyingAnimation;
     var ticker = function(tf) {
-      var player = underlyingPlayer._wrapper;
-      if (player.playState == 'pending') return;
+      var animation = underlyingAnimation._wrapper;
+      if (animation.playState == 'pending') return;
 
-      if (!player.source)
+      if (!animation.effect)
         return;
       if (tf == null) {
-        player._removePlayers();
+        animation._removeChildren();
         return;
       }
     };
 
-    underlyingPlayer = scope.timeline.play(new scope.Animation(null, ticker, group._timing));
-    return underlyingPlayer;
+    underlyingAnimation = scope.timeline.play(new scope.KeyframeEffect(null, ticker, group._timing));
+    return underlyingAnimation;
   };
 
-  scope.bindPlayerForGroup = function(player) {
-    player._player._wrapper = player;
-    player._isGroup = true;
-    scope.awaitStartTime(player);
-    player._constructChildren();
-    player._setExternalPlayer(player);
+  scope.bindAnimationForGroup = function(animation) {
+    animation._animation._wrapper = animation;
+    animation._isGroup = true;
+    scope.awaitStartTime(animation);
+    animation._constructChildren();
+    animation._setExternalAnimation(animation);
   };
 
   scope.groupChildDuration = groupChildDuration;
 
+  // Alias GroupEffect & SequenceEffect to AnimationGroup & AnimationSequence respectively, to
+  // support old constructors (Animation*) for a deprecation period. Should be removed after 23 June
+  // 2015.
+  window.AnimationSequence = function() {
+    if (shared.isDeprecated('window.AnimationSequence', '2015-03-23', 'Use window.SequenceEffect instead.')) {
+      return;
```

Comment:
All I have to do is change shared.isDeprecated to shared.deprecated, with exactly the same arguments, right?